src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,103 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final int GROW_INDEX = 2;
+    private Node<K,V>[] table;
+    private int size;
+    private int threashold;
+    private Node<K,V> newNode;
+    private Node<K,V> boofNode;
+    private int index;
```

Comment:
should these variables belong to class or maybe we can create them locally inside the methods where we use them?